spec/factories/post.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :post do
```

Comment:
Factory girl generatorlere bak. Bunlari elle girmene gerek yok. Factory girl` in avantaji o zaten.